src/Game.java

```diff
@@ -0,0 +1,91 @@
+import java.io.BufferedReader;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+
+public class Game {
+    private WordsDatabase database = new WordsDatabase();
+    private Scanner sc = new Scanner(System.in);
+
+    public void hangmanGame() {
+        final int EXIT = 0;
+        final int PLAY = 1;
+        String fileName = "words.txt";
+        try (
+                var fileReader = new FileReader(fileName);
+                var bufferedReader = new BufferedReader(fileReader);
+        ) {
+
+            while (bufferedReader.readLine() != null) {
+                database.addWord(bufferedReader.readLine());
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
+
+        database.addWord("drzewo");
+        database.addWord("zamek");
+        database.addWord("kwiatek");
+        database.addWord("programowanie");
+        boolean isException = true;
+        while (isException) {
+            try {
+                int option = -1;
+                while (option != EXIT) {
+
+                    System.out.println("Dostępne opcję:");
+                    System.out.println("0 - wyjście z programu.");
+                    System.out.println("1 - graj.");
+                    option = sc.nextInt();
+                    sc.nextLine();
+
+                    switch (option) {
+                        case EXIT:
+                            System.out.println("Koniec gry.");
+                            sc.close();
+                            break;
+                        case PLAY:
+                            revealChar();
```

Comment:
This method name doesnt make sense with its implementation. Since you are actually starting a hangman game in this method.